src/strands/multiagent/a2a/server.py

```diff
@@ -176,16 +176,20 @@ def agent_skills(self, skills: list[AgentSkill]) -> None:
         """
         self._agent_skills = skills
 
-    def to_starlette_app(self) -> Starlette:
+    def to_starlette_app(self, **kwargs: Any) -> Starlette:
```

Comment:
In case there's something else we eventually want to pass here, we think it makes more sense to pass this in explicitly as a dictionary that is passed into each app.  For instance, something like:

```python
def to_starlette_app(self, *, app_args: dict[str, Any] | None = None):
    ...
    A2AStarletteApplication(...).build(**app_args)
    ...
```

Naming wise, `app_kargs` or `app_config`  `build_input` might make sense, depending on the naming of what `build` takes.